public/locales/en-US/grafana.json

```diff
@@ -3632,6 +3632,7 @@
       "errored-resource-count": "Errors",
       "page-loading": "Loading...",
       "rebuild-snapshot": "Rebuild snapshot",
+      "show-errors": "highlight errors",
```

Comment:
A nit, but maybe something like "Only view errors" would be more true to what's it is actually doing, which hides everything else that's not an error